fuzz/run_fuzz.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2020, Arm Limited, All Rights Reserved
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#          http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+set -e
+set -x
+
+# Start TPM server
+tpm_server &
+sleep 5
+tpm2_startup -c -T mssim
+
+# Find PKCS 11 slot number
+CONFIG_PATH="run_config.toml"
+../tests/per_provider/provider_cfg/pkcs11/find_slot_number.sh $CONFIG_PATH
+
+# Create corpus if it doesn't exist
+mkdir -p corpus/fuzz_service
+cp init_corpus/* corpus/fuzz_service
+
+set +e
+
+while [ true ]
+do
+    # Run fuzzer
+    cargo +nightly fuzz run fuzz_service
+
+    # Notify about crash
+    echo "Here we'd ping the webhook to notify"
```

Comment:
As we currently do not notify the crash, would it be easy to read the logs to find the crashes that happened? Could it be easier, for now, to remove the `while true` loop and exit at the first crash?

Reply:
would be, but why not continue? the crash data (i.e. the input that triggered the crash) is saved by the fuzzer anyway. So far the only crash that happened was because the fuzzer kept using up more and more memory and died when it reached its limit - not exactly a meaningful crash

Reply:
That's good then! As long as it is easy to know when a crash happenened and with what data.

Reply:
It doesn't only record crashes, it also records slow runs